lsstseries/ensemble.py

```diff
@@ -24,7 +25,7 @@ def tap_token(self, token):
         """
         self.token = token
 
-    def query_tap(self, query, maxrec=None):
+    def query_tap(self, query, maxrec=None,debug=True):
```

Comment:
Style: Add a space after the comma.

Reply:
I concour. Linter run is needed.

Reply:
Ran black on the three main files (ensemble, timeseries, stetsonj), first time using it but will be using it going forward so hopefully that will lower the frequency of my formatting issues 😅